src/style.css

```
@@ -0,0 +1,100 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+body {
+  margin: 0 5px;
+}
+
+.big__form {
+  margin: 20px 0;
+}
+
+.big__form__item {
+  position: relative;
+  width: 100%;
+  height: 70px;
+  text-indent: 20px;
+  font-family: Avenir-Book, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.big__form__item::placeholder {
+  position: absolute;
+  background-image: url(images/Search.svg);
+  top: 25px;
+  left: 36px;
+  background-repeat: no-repeat;
+  padding-left: 17px;
+}
+
+.big__form__item:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.big__form__item:focus {
+  font-family: Avenir-Heavy, sans-serif;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.small__form__item {
+  position: relative;
+  width: 100%;
+  height: 42px;
+  font-family: Avenir-Book, sans-serif;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  opacity: 0.27;
+}
+
+.small__form__item::placeholder {
+  position: absolute;
+  background-image: url(images/Search.svg);
+  background-size: 13px;
+  top: 12px;
+  left: 23px;
+  background-repeat: no-repeat;
+  padding-left: 29px;
+}
+
+.small__form__item:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  opacity: 1;
+}
+
+.small__form__item:focus {
+  font-family: Avenir-Heavy, sans-serif;
+  /* font-weight: 900; */
```

On focus, there should be bold text
```suggestion
  font-weight: 900;
```